<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>668</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="createTfRecords">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>550</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>CreateTfRecords</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ctgListComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>New Item</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>New Item2</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Category</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="addctgButton">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>100</y>
      <width>111</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Category</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="addCtglineEdit">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>50</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rmvctgButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove Category</string>
    </property>
   </widget>
   <widget class="QLabel" name="imgprvlabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>591</width>
      <height>400</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QListWidget" name="imgListWidget">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>341</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="saveAnnotsBtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>70</y>
      <width>99</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>SaveAnnots</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="imgResize">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>550</y>
      <width>131</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>ResizeImages</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="tfRecordProgress">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>590</y>
      <width>241</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <zorder>ctgListComboBox</zorder>
   <zorder>label</zorder>
   <zorder>addctgButton</zorder>
   <zorder>addCtglineEdit</zorder>
   <zorder>rmvctgButton</zorder>
   <zorder>imgprvlabel</zorder>
   <zorder>imgListWidget</zorder>
   <zorder>saveAnnotsBtn</zorder>
   <zorder>createTfRecords</zorder>
   <zorder>imgResize</zorder>
   <zorder>tfRecordProgress</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1060</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
